odxtools/diaglayer.py

```diff
@@ -170,6 +200,11 @@ def _finalize_init(self, odxlinks: OdxLinkDatabase) -> None:
         #####
         self._communication_parameters = self._compute_available_commmunication_parameters()
 
+        #####
+        # resolve all SNREFs. TODO: We allow SNREFS to inherited
```

Comment:
typo: to be inherited

Reply:
I wanted to say that it is allowed to specify an SNREF to an object that was inherited. I reformulated that a bit: 6e9ce255aed8f684098f86fb81063e65b030a573, thanks bringing this up.